apps/ui/components/search/SingleSearchForm.tsx

```diff
@@ -121,6 +126,10 @@ export function SingleSearchForm({
   const form = useForm<FormValues>({
     values: formValues,
   });
+  const accessTypeOptions = Object.values(AccessType).map((value) => ({
+    value,
+    label: t(`reservationUnit:accessTypes.${value}`),
+  }));
```

Comment:
`MULTIVALUED` is not wanted here

![image](https://github.com/user-attachments/assets/86b6211d-e46e-4d4e-98a0-b648fcdb7bc8)

Reply:
Deprecated: it doesn't snow anymore, after update to back end `AccessType`s.